src/compiler/diagnosticMessages.json

```diff
@@ -1834,6 +1830,10 @@
         "category": "Error",
         "code": 1544
     },
+    "'\\k' is only available outside character class.": {
+        "category": "Error",
+        "code": 1545
+    },

```

Comment:
Just to confirm, a new message can’t take the place of an old, unused one right?
(TS1513 never actually appears as TS1161 would have been emitted at the same position if a RegExp is unterminated, so replacing the message shouldn’t cause any problems.)